src/main/java/org/dependencytrack/persistence/jdbi/mapping/VulnerabilityPolicyRowMapper.java

```diff
@@ -0,0 +1,57 @@
+package org.dependencytrack.persistence.jdbi.mapping;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsonorg.JsonOrgModule;
+import org.dependencytrack.policy.vulnerability.VulnerabilityPolicy;
+import org.dependencytrack.policy.vulnerability.VulnerabilityPolicyAnalysis;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+import java.sql.Array;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.Optional;
+
+public class VulnerabilityPolicyRowMapper implements RowMapper<VulnerabilityPolicy> {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper().registerModule(new JsonOrgModule());
```

Comment:
`RowMapperUtil` has an `ObjectMapper` that you can reuse: https://github.com/DependencyTrack/hyades-apiserver/blob/b4454defbd8c0820c1be9f424c46a51da92bc139/src/main/java/org/dependencytrack/persistence/jdbi/mapping/RowMapperUtil.java#L26